ktor-client/ktor-client-tests/js/src/io/ktor/client/tests/utils/Assertions.kt

```diff
@@ -11,6 +11,14 @@ import io.ktor.util.*
  */
 @InternalAPI
 actual inline fun <reified T : Throwable> assertFailsAndContainsCause(block: () -> Unit) {
+    assertFailsWithJsSupported<T>(block)
+}
+
+/**
+ * Asserts that a [block] fails with a specific exception of type [T] being thrown.
+ */
+@InternalAPI
+actual inline fun <reified T : Throwable> assertFailsWithJsSupported(block: () -> Unit) {
```

Comment:
Consider using a different name here: `assertFailsWith` should be enough

Reply:
Yes, I fixed.